src/scenes/LoadingScene.ts

```diff
@@ -92,6 +92,10 @@ export class LoadingScene extends Phaser.Scene {
   };
 
   private changeScene = () => {
-    this.scene.start('MainMenuScene');
+    if (shouldSkipIntro()) {
```

Comment:
Let's add another env variable for that intro. When I'll be developing the intro I want to skip the bodging bear intro, but not the history intro (which I'm developing 😉)

Reply:
;)

Reply:
what a shitty PR